Prevention Act;Prevents sandmining, scamming and trapping.html

```diff
@@ -0,0 +1,24 @@
+<h2>Prevention</h2>
+<h3>Sandmining</h3>
+<p>
+    Sandmining is not allowed, if a player sandmines the military may be deployed
+    and if it’s severe the criminal may be given a fine of 1g per chunk
```

Comment:
It's probably best to say up to 1G per chunk, it's good to have room to decrease the fine in cirtain circumstances.
```suggestion
    and if it’s severe the criminal may be given a fine of up to 1g per chunk
```